discovery/seata-discovery-nacos/src/main/java/org/apache/seata/discovery/registry/nacos/NacosRegistryServiceImpl.java

```diff
@@ -194,8 +194,9 @@ public List<InetSocketAddress> lookup(String key) throws Exception {
                                     .map(eachInstance -> new InetSocketAddress(eachInstance.getIp(), eachInstance.getPort()))
                                     .collect(Collectors.toList());
                             CLUSTER_ADDRESS_MAP.put(clusterName, newAddressList);
-
-                            removeOfflineAddressesIfNecessary(transactionServiceGroup, clusterName, newAddressList);
+                            if (StringUtils.isNotEmpty(transactionServiceGroup)) {
```

Comment:
I think we should solve the problem of why the current transaction group is empty, instead of simply adding an empty judgment